src/components/actionBar/ActionBar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ROUTE_INDEX } from '@/router/routes';
+
+import LogoutButton from './LogoutButton';
+import MainButton from './MainButton';
+import PublishButton from './PublishButton';
+
+const ActionBar = ({ route }) => (
+  <div className='action-bar'>
+    <PublishButton />
+    <LogoutButton />
+    {route !== ROUTE_INDEX && <MainButton />}
```

Comment:
Why not use a `<Link />` here? Even better: with the `NavLink` you could use an active class name which hides the button when the index page is open